src/main/java/com/services/OrganizationService.java

```diff
@@ -78,10 +84,11 @@ public void deleteOrg (int id){
             throwable.printStackTrace();
         } catch (ClassNotFoundException e) {
             e.printStackTrace();
+            throw new NotFoundException("Organization not found for id:" + id);
         }
     }
 
-    public Organization updateOrg (Organization organization) {
+    public Organization updateOrg (Organization organization) throws NotFoundException {
```

Comment:
the method signature says it throws a Not Found exception, but the method never throws.